cmd/node-canary/main.go

```diff
@@ -253,6 +253,7 @@ func makeNodeConfig() (*params.NodeConfig, error) {
 		nodeConfig.LogFile = *logFile
 	}
 
+	nodeConfig.NoDiscovery = true
```

Comment:
Is it needed? `NewNodeConfigWithDefaults` sets `c.NoDiscovery = true` now and `WithFleet` option is not set.

Reply:
Yeah, it's only making sure that it's enabled, but we can remove it.